src/data_provider/src/osinfo/sysOsParsers.cpp

```diff
@@ -371,17 +359,25 @@ bool HpUxOsParser::parseUname(const std::string& in, nlohmann::json& output)
     return ret;
 }
 
+bool AlpineOsParser::parseFile(std::istream& in, nlohmann::json& output)
+{
+    constexpr auto PATTERN_MATCH{R"((?:[0-9]+\.)?(?:[0-9]+\.)?(?:[0-9]+))"};
+    output["os_name"] = "Alpine Linux";
+    output["os_platform"] = "alpine";
+    return findVersionInStream(in, output, PATTERN_MATCH);
+}
+
 bool MacOsParser::parseSwVersion(const std::string& in, nlohmann::json& output)
 {
     constexpr auto SEPARATOR{':'};
-    static const std::map<std::string, std::string> KEY_MAP
+    static const std::vector<std::pair<std::string, std::string>> KEY_MAPPING
```

Comment:
The changes made to this function, while elementary, exceed the original scope of the issue.